transifex_catalogs/en.json

```diff
@@ -67,7 +67,10 @@
   "exchanges_payment_title": "Decred Payment Processors",
   "exchanges_payment_description": "Solutions for online merchants to start accepting Decred (DCR) as payment. Payment processors provide a quick and easy integration to accept cryptocurrencies through e-commerce platforms.",
   "exchanges_subtitle": "Decred on Exchanges",
-  "exchanges_showall": "Show all",
+  "exchanges_showall": "Exchanges",
```

Comment:
I may be incorrect on this, but I believe these new keys needed to be added to transifex files for all languages, not just `en`. @peterzen could you please confirm?

Reply:
It's correct as it is - the other languages will get populated after they've been translated in Transifex. This file will have to be pushed to TF once the PR is approved, using `bin/transifex_push.sh`

Reply:
Ok